nixos/modules/services/misc/gogs.nix

```diff
@@ -217,7 +217,6 @@ in
           sed -e "s,#secretkey#,$KEY,g" \
```

Comment:
TODO(tangentially related to this PR): we should use replace-secret instead of sed to not leak secrets in process listings.